### ISSUE TITLE
release: v20.1.1 (switched to a different sha: extraordinary release delay)

### ISSUE THREAD

**Issue by jlinder** [cockroachdb/cockroach#49201]
Candidate SHA: c80790db80a40d2ffa1dc29249662eeead219f11
Deployment status: Qualifying
Qualification Suite: https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_ReleaseQualification/1952317
Nightly Suite: https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_NightlySuite/1952337
 
Admin UI for Qualification Clusters:
- Old Version: http://jamesl-release-v2010-9d456b-0001.roachprod.crdb.io:26258/#/overview/list
- New Version: http://jamesl-release-v2011-c80790-0001.roachprod.crdb.io:26258/#/overview/list
 
[Release process checklist](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process)
 
Prep date: `May 18, 2020`
- [x] [Pick a SHA](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-PickingaSHA)
  - [x] fill in `Candidate SHA` above
  - [x] email thread on releases@
- [x] [Tag the provisional SHA](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-TagtheprovisionalSHA)
- [x] [Publish provisional binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Publishtheprovisionalbinaries)
- [x] Ack security@ on the generated Stackdriver Alert to confirm these writes were part of a planned release (Just reply on the email received alert email acking that this was part of the release process)
 
Release Qualification
- [x] [Check binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Checkbinaries)
- [x] [Deploy to test clusters](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-DeploytoTestClusters)
- [x] [Verify node crash reports](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Verifynodecrashreportsappearinsentry.io)
- [x] [Start release qualification suite](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-RuntheSuiteofNightlyRoachtests)
- [x] [Start nightly suite](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-RuntheSuiteofNightlyRoachtests)
- [x] fill in all WIP elements above (Nightly Suite, Admin UI, etc.)
 
One day after prep date:
- [ ] [Get signoff on roachtest failures](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-RoachtestFailureSignoff)
 
- [ ] [Keep an eye on clusters](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/328859690/Release+Qualification#Monitoring-the-Qualification-Clusters) until release date. Do not proceed below until the release date.
 
Release date: `WIP fill in date: usually Monday WIP`
- [ ] [Check cluster status](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/328859690/Release+Qualification#Monitoring-the-Qualification-Clusters)
- [ ] [Tag release](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Tagtherelease)
- [ ] [Bless provisional binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Blesstheprovisionalbinaries)
 
- For production or stable releases in the latest major release series
  - [ ] Update the [Homebrew Formula](https://github.com/cockroachdb/homebrew-tap/blob/master/Formula/cockroach.rb)
  - [ ] Update the image tag in our [Orchestrator configurations](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Orchestratorconfigurations)
  - [ ] Update the image tag in Helm's [Orchestrator configurations](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Orchestratorconfigurations)
 
- For production or stable releases
  - [ ] Announce version to the [registration cluster](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-AnnounceVersionToRegCluster)
 
- [ ] Update docs
- [ ] External communications for release
- [ ] Clean up provisional tag from repository

**Comment by jlinder** [comment]
An extraordinary release delay request came in and a new sha is being chosen. The new issue: https://github.com/cockroachdb/cockroach/issues/49259
